diff --git a/dist/connection.js b/dist/connection.js
index e98f42e8077944d297976de4a7c61ab7d32bef82..8d90acdb9e3f56daf4900d92c04397311e74ed0e 100644
--- a/dist/connection.js
+++ b/dist/connection.js
@@ -201,6 +201,7 @@ class Connection extends eventemitter3_1.EventEmitter {
         }
         this._socket = new net_1.Socket();
         this._socket.setEncoding('utf-8');
+        this._socket.setKeepAlive(true);
         this._socket.on('data', (data) => {
             try {
                 this._processIncomingData(data);
